src/v2/Apps/Artwork/Components/ArtworkSidebar/CreateArtworkAlertSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Separator, Spacer, Flex, Button, BellIcon, Text } from "@artsy/palette"
+
+export const CreateArtworkAlertSection: React.FC = () => {
+  return (
+    <>
+      <Separator />
+      <Spacer m={2} />
+      <Flex
+        flexDirection="row"
+        py={1}
+        alignItems="center"
+        justifyContent="space-between"
+      >
+        <Flex flex={1} justifyContent="flex-start">
+          <Text variant="xs" color="blue100">
+            Be notified when a similar piece is available
+          </Text>
+        </Flex>
+        <Flex flex={1} justifyContent="flex-end">
+          <Button focus variant="secondaryOutline" size="small">
+            <BellIcon mr={0.5} fill="currentColor" />
+            <Text variant="xs">Create Alert</Text>
+          </Button>
```

Comment:
I think this will be replaced by `https://github.com/artsy/force/blob/5f4250cbd45fc81edeaf35e0c32aaee4ec7d2692/src/v2/Components/SavedSearchAlert/Components/SavedSearchCreateAlertButton.tsx` but since this ticket doesn't include the modal went with a simpler approach for now